src/main/java/org/eln2/mc/common/content/Ground.kt

```diff
@@ -0,0 +1,73 @@
+package org.eln2.mc.common.content
+
+import net.minecraft.resources.ResourceLocation
+import net.minecraft.world.phys.Vec3
+import org.ageseries.libage.sim.electrical.mna.Circuit
+import org.eln2.mc.client.render.PartialModels
+import org.eln2.mc.client.render.foundation.BasicPartRenderer
+import org.eln2.mc.common.cells.CellRegistry
+import org.eln2.mc.common.cells.foundation.CellBase
+import org.eln2.mc.common.cells.foundation.CellPos
+import org.eln2.mc.common.cells.foundation.Conventions
+import org.eln2.mc.common.cells.foundation.objects.ElectricalComponentInfo
+import org.eln2.mc.common.cells.foundation.objects.ElectricalObject
+import org.eln2.mc.common.cells.foundation.objects.ResistorBundle
+import org.eln2.mc.common.cells.foundation.objects.SimulationObjectSet
+import org.eln2.mc.common.parts.foundation.CellPart
+import org.eln2.mc.common.parts.foundation.ConnectionMode
+import org.eln2.mc.common.parts.foundation.IPartRenderer
+import org.eln2.mc.common.parts.foundation.PartPlacementContext
+import org.eln2.mc.common.space.RelativeRotationDirection
+
+/**
+ * The ground object is simply a bundle of resistors, with one grounded pin.
+ * The ungrounded pin is exported to other Electrical Objects.
+ * */
+class GroundObject : ElectricalObject() {
+    private val resistors = ResistorBundle(0.01)
```

Comment:
This resistor constant should be pulled from `libage` electrical constants for wire resistance.